packages/components/tests/dummy/app/templates/components/table.hbs

```diff
@@ -21,43 +26,89 @@
     <code class="dummy-code">thead</code>
     of the table. It yields within a single
     <code class="dummy-code">tr</code>
-    element that is meant to represent column headers.</p>
+    element that contains column headers.</p>
   <h4 class="dummy-h4">:body named block</h4>
   <p class="dummy-paragraph">This represents the
     <code class="dummy-code">tbody</code>
-    of the table. It yields
-    <code class="dummy-code">n</code>
-    number of times where
-    <code class="dummy-code">n</code>
-    is equal to the length of the
-    <code class="dummy-code">@model</code>
-    arg provided to Table. The yield happens directly in the
-    <code class="dummy-code">tbody</code>
-    element to allow for multiple
-    <code class="dummy-code">tr</code>
-    elements for a single record. The yielded value is the exact record from the
-    <code class="dummy-code">@model</code>
-    array.</p>
+    of the table.
+  </p>
```

Comment:
We seem to be exposing [named blocks as part of the 'Component API' section](https://hds-components-9g9m1i02q-hashicorp.vercel.app/utilities/disclosure) as `<:head> named block` and `<:body> named block`